backend/endpoints/feeds.py

```diff
@@ -161,12 +161,13 @@ async def extract_titledb(roms: list[Rom]) -> dict[str, TinfoilFeedTitleDBSchema
             TinfoilFeedFileSchema(
                 url=str(
                     request.url_for(
-                        "get_rom_content", id=rom.id, file_name=rom.file_name
+                        "get_rom_content", id=rom.id, file_name=file.get('filename')
                     )
                 ),
-                size=rom.file_size_bytes,
+                size=rom.files[0].get('size'),
             )
-            for rom in roms
+            for rom in roms if rom.files and len(rom.files) > 0
+            for file in rom.files
```

Comment:
I don't think we want to expose all files to Tinfoil. Ideally, we could only include the ones with a valid extension that Tinfoil understands (in case users maintain other kind of resources within the rom folder).